components/bl-transitions-component/README.md

```diff
@@ -0,0 +1,20 @@
+# Transition
+
+Transition is a component of the Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). Transition allows you to add a smooth component mounting animation.
+
+The component based on external [Transition](https://mui.com/material-ui/transitions/).
+
+## Properties
+
+| Property          | Type                                                             | Default Value  | Logic | Data Binding | UI Setting | Description                                     |
+|-------------------|------------------------------------------------------------------|----------------|-------|--------------|------------|-------------------------------------------------|
+| variants          | Select ["collapse-top", "collapse-left", "fade", "grow", "zoom"] | "collapse-top" |       | NO           | YES        | Allows to determine variant of transition       |
+| animationDuration | Number                                                           | 300            |       | NO           | YES        | Allows to determine duration of transition (ms) |
```

Comment:
I think we should use labels like this: `Variants`, `Animation Duration`. In general, we make it for customers, not for developers.